source/change-streams/change-streams.rst

```diff
@@ -483,27 +477,11 @@ Once a ``ChangeStream`` has encountered a resumable error, it MUST attempt to re
 
 - Perform server selection.
 - Connect to selected server.
-- If the ``ChangeStream`` has a cached ``postBatchResumeToken`` and has returned all documents in the most recent batch (or the most recent batch was empty):
-
-  - The driver MUST set ``resumeAfter`` to the cached ``postBatchResumeToken``.
+- If there is a cached ``resumeToken``:
```

Comment:
this is how this comment looks like in my browser:
![image](https://user-images.githubusercontent.com/44870738/56134644-b00f3d80-5f97-11e9-86ed-407b04438dc7.png)

Reply:
Looks like this has been fixed since this comment was posted.